.gitignore

```diff
@@ -1,3 +1,4 @@
 .vs/
 bin/
-obj/
\ No newline at end of file
+obj/
+.csproj.user
```

Comment:
Incorrect rule. It should be `*.csproj.user`, otherwise it won't ignore it. Rule actually should be `*.user`